pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -186,13 +186,19 @@ func New(options Options) Index {
 	)...)
 	authzPolicies := kclient.NewDelayedInformer[*securityclient.AuthorizationPolicy](options.Client,
 		gvr.AuthorizationPolicy, kubetypes.StandardInformer, configFilter)
+	// Start with a.stop to ensure the informer respects the index's stop channel
+	authzPolicies.Start(a.stop)
```

Comment:
Hmm, these got started before this PR; otherwise the feature wouldn't work. What was starting them and why isn't that caller's stop sufficient?